pkg/api/handlers/utils/handler.go

```diff
@@ -59,3 +62,18 @@ func FilterMapToString(filters map[string][]string) (string, error) {
 	}
 	return string(f), nil
 }
+
+func getVar(r *http.Request, k string) string {
+	val := mux.Vars(r)[k]
+	safeVal, err := url.PathUnescape(val)
+	if err != nil {
+		logrus.Error(errors.Wrapf(err, "failed to unescape mux value %s", val))
```

Comment:
k and url would make searching journald possible.  Are those included in the err you are wrapping?